### DISCUSSION TITLE
Best platform to sponsor fabricjs?

### DISCUSSION THREAD

**Discussion by hamzawain7** [fabricjs/fabric.js#6753] [General]
What method do you prefer? Is your patreon (https://www.patreon.com/fabricJS) valid?

**Comment by asturur** [comment]
Yes is valid, but for now github sponsor is a better way, because is still commission free.
Also actually Patreon being US only requires a lot of forms to be filled to retrieve the money and the use of paypal, that will add extra fee on top of the patreon fee.

It must be noted tho, that we do not have direct cost for maintain @fabricjs right now.
The work is done in the free time and the website is paid by the original author.

Said so we have a guy, @nimworks, that started a new fabricJS website and he works as a freelancer.
I would not mind turning to him the current donations to incetivate his work.
